### DISCUSSION TITLE
Concerns about new Alt+Tab behavior

### DISCUSSION THREAD

**Discussion by vbuell** [elementary/gala#1338] [Ideas]
### Problem

Personally I prefer the way Alt+Tab worked before. Previous behavior was inline with the concept of simplicity, where instead of creating new distracting popups Elementary just was reusing the existing controls on screen. The new switcher IMHO is a step back for elementary as it tries to mimic old ideas and be just like others. Not saying that others are bad, but the elementary's solution was elegant and eye catchy, and that's the reason it found its great nieche for people who likes simplicity.

Some similar comments or Reddit: https://www.reddit.com/r/elementaryos/comments/r5x3qr/new_alttab_is_late_but_it_is_awesome/

### Proposal

We already have the tasks widget on bottom; if some people prefer to see it in a center of screen, we could dynamically scale it up and slide it to a center while Alt key is pressed. And put it back to the bottom when Alt is released. It's very similar to the new switcher (clasical popup) but less intrusive to existing users and it combines both worlds.

I can't be 100% sure that this proposal will work for all users, but at least it solves some issues from the list above, and doesn't introduce new components. Would be glad to hear any feedback. Also please let me know is there a better place to start this discussion (forum thread, etc) rather than opening a feature request.

**Comment by cassidyjames** [comment]
@vbuell I've converted this from issue #1337 to a discussion, in case you're wondering why the old issue looks closed (I know on mobile it can be less clear).

**Comment by pushqrdx** [comment]
> but the elementary's solution was elegant and eye catchy

Application switchers shouldn't be eye catchy, that was the one of the worst aspects of the old design, it was distracting. New design works and has been working for desktop users for years, if it ain't broke don't fix it

**Comment by chbybnwr** [comment]
Would love to contribute just to bring back the old switcher. It wouldn't be the default, users have to opt-in.
Probably will do it the hard way, possibly will give up midway, never did system level development before.
Just need the green light, to make sure that it will be merged on the slim chance that it does work, or if it is even possible?

**Reply by lenemter** [reply]
This probably will not get merged because old window switcher had a dependency on plank which doesn't work on Wayland. There's a new Wayland compatible dock in early development but I don't think it can be used as a window switcher.

If you really want the old window switcher you can write a plugin for Gala which isn't intended to be merged and users must install it manually to use it. You can brouse through old code and take code from there i guess :)

**Reply by typomedia** [reply]
How to activate my new `usr/lib/x86_64-linux-gnu/gala/plugins/libgala-switcher.so`?

    gsettings list-recursively io.elementary.gala
    No schema »io.elementary.gala« found
